crates/cli-tools/src/lib.rs

```diff
@@ -17,6 +17,7 @@
 //! This library is also used for the internal maintenance CLI of Wasefire called xtask.
 
 #![feature(async_fn_track_caller)]
+#![feature(never_type)]
```

Comment:
Looks like a left-over, but should be useful again when fixing the other comment.